src/test/java/uk/gov/hmcts/probate/service/template/pdf/SolicitorCoversheetPDFDecoratorTest.java

```diff
@@ -41,6 +45,7 @@ public void setup() {
     
     @Test
     public void shouldNotProvideAdditionalDecoration() {
+        when(authenticatedTranslationBusinessRuleMock.isApplicable(caseDataMock)).thenReturn(false);
```

Comment:
Perhaps we should set ALL business rules to return false in the setup method, then only set the true returns in the specific tests?

Reply:
updated thanks